pkg/util/converter/prom.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
```

Comment:
Unrelated, but I have the impression that we should have our own version of this file, specially for decoupling.

Reply:
Yes, definitely, there's already some discussion about that.